extensions/add-global-attributes.js

```diff
@@ -4,33 +4,83 @@
  *    - require: ./extensions/add-global-attributes.js
 */
 
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+const _ = require('lodash');
+
+const ATTRIBUTES_PATH = 'modules/ROOT/partials/';  // Default path within the 'shared' component
+
 module.exports.register = function ({ config }) {
-  const yaml = require('js-yaml');
   const logger = this.getLogger('global-attributes-extension');
-  const chalk = require('chalk')
-  const _ = require('lodash');
+  const chalk = require('chalk');
+
+  /**
+   * Load global attributes from a specified local file if provided.
+   */
+  function loadLocalAttributes(siteCatalog, localAttributesFile) {
+    try {
+      const resolvedPath = path.resolve(localAttributesFile);
+      if (!fs.existsSync(resolvedPath)) {
+        logger.warn(`Local attributes file "${localAttributesFile}" does not exist.`);
```

Comment:
not for now, but we might consider adding a new npm goal with verbose logging.

This one for example, could be omitted in most cases.

Reply:
I used `warn` here because it fits the descriptions of the Antora log levels: https://docs.antora.org/antora/latest/playbook/runtime-log-level/

We could have a cut-down playbook that has `fatal` logs on only. Or we could document how to do that in a wiki so anyone can customize the local playbook as necessary.

We can discuss more after next week 😄